core/src/main/java/org/stellar/anchor/util/Sep1Helper.java

```diff
@@ -1,26 +1,39 @@
 package org.stellar.anchor.util;
 
+import static org.stellar.anchor.util.Log.*;
+
 import com.moandjiezana.toml.Toml;
 import java.io.IOException;
-import java.net.URL;
+import org.stellar.anchor.api.exception.InvalidConfigException;
 
 public class Sep1Helper {
   public static TomlContent readToml(String url) throws IOException {
-    return new TomlContent(new URL(url));
+    try {
+      String tomlValue = NetUtil.fetch(url);
+      return new TomlContent(tomlValue);
+    } catch (IOException e) {
+      String obfuscatedMessage =
```

Comment:
Exception message is swallowed, we should log it. Same for method below

Reply:
whoops i think i confused myself. meant to log the exception. :) fixed that. thanks